frontend/src/components/RisLawPreview.spec.ts

```diff
@@ -10,4 +10,123 @@ describe("RisLawPreview", () => {
     expect(screen.getByText("Test Title")).toBeInTheDocument()
     expect(screen.getByText("Test Title").className).contain("longTitle")
   })
+
+  test("should emit click event", async () => {
+    const handler = vi.fn()
+
+    render(RisLawPreview, {
+      props: {
+        content:
+          "<div><span class='longTitle'>Test Title</span><div class='akn-mod' data-eId='hauptteil-1_art-1_abs-1_untergl-1_listenelem-1_inhalt-1_text-1_ändbefehl-1' data-GUID='148c2f06-6e33-4af8-9f4a-3da67c888510'>MOD</div></div>",
+      },
+      attrs: {
+        "onClick:akn:mod": handler,
+      },
+    })
+
+    await waitFor(() => screen.getByRole("button"))
+    await fireEvent.click(screen.getByRole("button", { name: "MOD" }))
```

Comment:
Question: Did you use `fireEvent` here for a specific reason? Usually we use [userEvent](https://testing-library.com/docs/user-event/intro/).

Reply:
didn't knew of that so I'll change it :)